__tests__/Relude_TreeZipper_test.re

```diff
@@ -662,6 +662,16 @@ describe("TreeZipper", () => {
     expect(actual) |> toEqual(expected);
   });
 
+  test("moveUpToTop maintains structure", () => {
```

Comment:
Not sure what to call the test or where to put it. Would appreciate some advice here :smile:

Reply:
This seems fine to me